vmware-base/src/com/cloud/hypervisor/vmware/mo/SnapshotDescriptor.java

```diff
@@ -166,12 +168,7 @@ public void removeDiskReferenceFromSnapshot(String diskFileName) {
             assert (false);
             s_logger.error("Unexpected exception ", e);
         } finally {
-            if (out != null) {
-                try {
-                    out.close();
-                } catch (IOException e) {
-                }
-            }
+            closeAutoCloseable(out, "buffered write for bytestream");
```

Comment:
Yes, due to the amount of tedious work I am choosing try-with-resource only when it is easy. It is in this case so I will change it, thanks @koushik-das